src/test/java/org/apache/sysds/test/functions/ternary/TernaryAggregateTest.java

```diff
@@ -41,10 +41,11 @@ public class TernaryAggregateTest extends AutomatedTestBase
 {
 	private final static String TEST_NAME1 = "TernaryAggregateRC";
 	private final static String TEST_NAME2 = "TernaryAggregateC";
+	private final static String TEST_NAME3 = "TernaryAggregatePow";
 	
 	private final static String TEST_DIR = "functions/ternary/";
 	private final static String TEST_CLASS_DIR = TEST_DIR + TernaryAggregateTest.class.getSimpleName() + "/";
-	private final static double eps = 1e-8;
+	private final static double eps = 1e-7;
```

Comment:
had to change 1e-7, because test failing locally on previous commits were the github tests passed (might be caused by local R version / ARM CPU ?)